products/db-knownprojects/bash/config.sh

```diff
@@ -32,26 +32,6 @@ function max_bg_procs {
     done
 }
 
-function import_private {
-  name=$1
-  version=${2:-latest} #default version to latest
-  version=$(mc cat spaces/edm-recipes/datasets/$name/$version/config.json | jq -r '.dataset.version')
-  echo "$name version: $version"
-  mc cp spaces/edm-recipes/datasets/$name/$version/$name.sql $name.sql
-  run_sql_file $name.sql
-  rm $name.sql
-}
-
-function import_public {
-  name=$1
-  version=${2:-latest}
-  version=$(curl -s https://nyc3.digitaloceanspaces.com/edm-recipes/datasets/$name/$version/config.json | jq -r '.dataset.version')
-  echo "$name version: $version"
-  curl -O https://nyc3.digitaloceanspaces.com/edm-recipes/datasets/$name/$version/$name.sql
-  run_sql_file $name.sql
-  rm $name.sql
-}
-
 function CSV_export {
```

Comment:
These exports should also be moved over! This csv export seems to be same as in shared utils. Same goes for shp export. Not sure at first glance about archive.

Above, urlparse and max_bg_processes can also be deleted

Reply:
thanks! turns out the whole config file can go!